content/community/casualconversation.md

```diff
@@ -18,7 +18,7 @@ subreddit: "CasualConversation"
 
 **What sets your community apart from others?**
 
-* {{< subreddit_link CasualConversation >}} is a subreddit for general chat about *almost* anything that you can think of. The main rules here are to "respect others" and "Don't take things too seriously." Mostly we just want people to have a friendly place to have conversations but we do have a [detailed rules page here](http://www.reddit.com/r/CasualConversation/wiki/rules). Oh we also have a Snoonet rules page that can be found at https://www.snoonet.org/ccrules.
+* {{< subreddit_link CasualConversation >}} is a subreddit for general chat about *almost* anything that you can think of. The main rules here are to "respect others" and "Don't take things too seriously." Mostly we just want people to have a friendly place to have conversations but we do have a [detailed rules page here](http://www.reddit.com/r/CasualConversation/wiki/rules). Oh we also have a Snoonet rules page that can be found at https://snoonet.org/ccrules.
```

Comment:
```
Oh we also have a Snoonet rules page that can be found at [here](/ccrules).
```

Reply:
Fixed in aae7a23f41e58728cdb2080121a3d8b2effc4c28